### PR TITLE
Fix rare randomly occurring test failure

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/moduledeployment/ModuleDeploymentTestSuite.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.jboss.as.test.integration.jca.moduledeployment;
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/moduledeployment/ModuleDeploymentTestSuite.java]: Copyright. :)

**Comment by n1hility** [comment]
Is that a new intermittent failure?

**Comment by ctomc** [comment]
@n1hility it is from the same "batch" of intermittent, all are related to JCA and they share parts of same test code.

**Comment by ctomc** [comment]
this was fixed by https://github.com/wildfly/wildfly-core/pull/659

**Pull request closed without merging**